### ISSUE TITLE
No formula editor displayed to students

### ISSUE THREAD

**Issue by mudrd8mz** [wiris/moodle-qtype_multianswerwiris#3]
The plugin description reads

> On the student's side, they will be able to introduce their answers by using a WYSIWYG formula editor and, if the teacher so specifies, they will have access to a WIRIS cas session to make some calculations.

I tried to use a SHORTANSWER subquestion with a variable value as the right answer. But to the student, there is no wiris input element displayed. Only the common HTML input field is displayed.

**Comment by manuelwiris** [comment]
Hi David, 

To use WIRIS editor in popup on a Cloze SHORTANSWER you should select "WIRIS editor in popup" Input Method in WIRIS Quizzes studio:

![image](https://cloud.githubusercontent.com/assets/10197199/22056653/690a5d4e-dd61-11e6-8f9c-4e36d8dce02b.png)

Best,

**Comment by mudrd8mz** [comment]
Yes I had that as I had been following http://www.wiris.com/en/quizzes/docs/moodle/manual/cloze

However I finally realised the missing WIRIS editor is caused by the PHP warning displayed (see reported issue #2). Compare the following screenshots.

Here I have debugging disabled:

![image](https://cloud.githubusercontent.com/assets/85831/22063784/02957074-dd81-11e6-8828-9942ad2d3b78.png)

Here I have debugging enabled and the PHP warning breaks the editor loading: 

![image](https://cloud.githubusercontent.com/assets/85831/22063812/2425bc08-dd81-11e6-92d0-fcd471ce7aa5.png)
